sagemathinc/cocalc
# Project uptime fix

# Description
This depends on https://github.com/sagemathinc/cocalc/pull/3209 ... i.e. this ticket should be merged afterwards.

The point here is to take the immediately available time info about when the project started instead of the time that's being set by the project. That way, the uptime info is available immediately.

Before:

![screenshot-cocalc com-2019 02 25-15-01-17](https://user-images.githubusercontent.com/207405/53342562-837f6380-390e-11e9-8c5e-293069ce852a.png)


After (taken on dev.cocalc.com, compiled today):

![screenshot-dev cocalc com-2019 02 25-15-00-22](https://user-images.githubusercontent.com/207405/53342567-8712ea80-390e-11e9-9a8e-4278099af6f0.png)


# Testing Steps
1. restart the project
2. check what the uptime says

# Relevant Issues

### [Checklist](https://github.com/sagemathinc/cocalc/wiki/PR-Checklist):

Front end:
